texar/torch/modules/pretrained/pretrained_bert.py

```diff
@@ -39,19 +39,35 @@ class PretrainedBERTMixin(PretrainedMixin, ABC):
     The BERT model was proposed in
     `BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding`_
     by `Devlin et al.` It is a bidirectional Transformer model pre-trained
-    on a large corpus. The available BERT models are as follows:
```

Comment:
(Edited a bit as below. Note to use nested list to avoid confusion):

Both standard BERT models and many variants are supported. Usually, you can specify the :attr:`pretrained_model_name` argument to pick which prtrained BERT model to use. All available categories of pretrained models (and names) include:

- **Standard BERT**:  proposed in (Devlin et al. 2018) `BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding`_.  A bidirectional Transfomer language model pre-trained on large text corpora. Available model names include:

  - ``bert-base-uncased``: 12-layer, 768-hidden, 12-heads, 110M parameters.
  - ``bert-large-uncased``: 24-layer, 1024-hidden, 16-heads, 340M parameters.
 ...

- **RoBERTa**: proposed in (Liu et al. 2019) `RoBERTa: A Robustly Optimized BERT Pretraining Approach`_. As a variant of the standard BERT model, RoBERTa trains for more iterations on more data with a larger batch size as well as other tweaks in pre-training. Differing from the standard BERT, the RoBERTa model does not use segmentation embedding.  Available model names include:

  - ``roberta-base``: RoBERTa using the BERT-base architecture, 125M parameters.
  - ``roberta-large``: RoBERTa using the BERT-large architecture, 355M parameters.